component/src/main/java/io/siddhi/extension/io/kafka/source/KafkaConsumerThread.java

```diff
@@ -72,20 +74,27 @@
     private KafkaSource.KafkaSourceState kafkaSourceState;
     private String[] requiredProperties;
     private int trpLength;
+    private SourceMetrics metrics;
+    private String groupId; //Needed for metrics
+//    private String streamId;
```

Comment:
Shall we remove this unused variable

Reply:
Removed the commented line. Thanks for pointing that out.